README.md

```diff
@@ -109,6 +109,34 @@ filter:{
 ```
 
 
+## If you have need custom filter on relation :
```

Comment:
If you need custom filter on relation : (For example array search in postgresql)